app.py

```diff
@@ -1,25 +1,51 @@
 import connexion
-from demo.routes import build_auth_code_flow
+from api.demo.routes import build_auth_code_flow
+from config.env import env
+from connexion.resolver import MethodViewResolver
 from flask import Flask
+from flask import request
+from flask_redis import FlaskRedis
 from flask_session import Session
+from flask_talisman import Talisman
 
+redis_mlinks = FlaskRedis(config_prefix="REDIS_MLINKS")
 
-def create_app() -> Flask:
-    connexion_app = connexion.FlaskApp(__name__, specification_dir="openapi/")
+
+def create_app(testing=False) -> Flask:
```

Comment:
📝 Not a bad solution to pass in "testing" as a kwarg to create_app, especially since we can set up a test fixture to only use the "testing=True" version of create_app.

Reply:
👌